webpack.config.js

```diff
@@ -14,6 +14,11 @@ const banner = `
   @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL v3.0
   `;
 
+const commitHash = require("child_process")
```

Comment:
This is breaking CI, and probably shouldn't be necessary, because the docker images already set that commit hash.

Reply:
Should we make this Docker-agnostic? What if running the frontend without Docker?

Reply:
The hash needs to be available during webpack build in order to write correct import paths into the generated js files. Why is it breaking CI? Maybe we can work around it

Reply:
Ahh I see, `git` is not available in the `node:alpine` image that is used for builds

Reply:
We could do one of the following:

1. Create a custom builder docker image which just installs git on top of node:alpine
2. Manually run `apk add git` as a first step before `yarn build` in .woodpecker.yml
3. Switch to a different base image which comes with git out of the box

Btw, option 2 is actually not very expensive at all, takes less than a second:

```
/ # time apk add git
(1/9) Installing ca-certificates (20230506-r0)
(2/9) Installing brotli-libs (1.0.9-r14)
(3/9) Installing libunistring (1.1-r1)
(4/9) Installing libidn2 (2.3.4-r1)
(5/9) Installing nghttp2-libs (1.53.0-r0)
(6/9) Installing libcurl (8.1.2-r0)
(7/9) Installing libexpat (2.5.0-r1)
(8/9) Installing pcre2 (10.42-r1)
(9/9) Installing git (2.40.1-r0)
Executing busybox-1.36.1-r0.trigger
Executing ca-certificates-20230506-r0.trigger
OK: 21 MiB in 26 packages
real	0m 0.69s
user	0m 0.27s
sys	0m 0.03s
```